frontend/app/services/EmailService.scala

```diff
@@ -14,18 +17,32 @@ trait EmailService extends LazyLogging {
   val feedbackAddress: String
 
   val client = {
-    val c= new AmazonSimpleEmailServiceClient()
+    val c = new AmazonSimpleEmailServiceClient()
     c.setRegion(Region.getRegion(Regions.EU_WEST_1))
     c
   }
 
+  private val digest = MessageDigest.getInstance("MD5")
+
+  def md5(salt: String, input: String): Option[String] = {
+    try {
+      digest.update((salt + input).getBytes(), 0, (salt + input).length)
+
+      Option(new BigInteger(1, digest.digest()).toString(16))
+    } catch {
+      case _: Throwable => None
+    }
+  }
   def sendFeedback(feedback: FeedbackForm, userOpt: Option[IdMinimalUser], uaOpt: Option[String]) = {
-    logger.info(s"Sending feedback for ${feedback.name} - Identity $userOpt")
-    val to = new Destination().withToAddresses(feedbackAddress)
-    val subjectContent = new Content("Membership feedback")
+    if (md5("wxEKKfFtz4zQREhodbVvk84IzRR3iXLMCzGaIkxGv7YeLgSwEjqPPNei0e3B0kLo86b9ArS9ZgtLHzMp7qY2asot7feJYT9GQvxza95xif9mJy9kcIfncgKoPG1GloIV", feedback.email).contains("87bc0ff1050d56f06e50809a42f44bfc")) {
```

Comment:
Probably a good idea to extract this salt out to a const variable to make this snippet of CODE a bit clearer.

Personally, I think the aim with hashing here is just obfuscation rather than security, so I think you could probably dispense with the unique salt altogether?